lib/ipmi_delloem.c

```diff
@@ -922,17 +922,17 @@ ipmi_lcd_get_info_wh(struct ipmi_intf * intf)
 {
 	IPMI_DELL_LCD_CAPS lcd_caps;
 	char lcdstring[IPMI_DELL_LCD_STRING_LENGTH_MAX+1] = {0};
-	int rc;
+	int rc = 0;
 	printf("LCD info\n");
 	if (ipmi_lcd_get_configure_command_wh(intf) != 0) {
-		return -1;
+		ERR_EXIT;
```

Comment:
You won't need this if you default to error. Just `goto out`.

Reply:
In this particular case, I could have also just done "goto exit" but specifically used the macro for consistency.